core/src/main/kotlin/org/evomaster/core/output/service/TestCaseWriter.kt

```diff
@@ -76,6 +82,10 @@ abstract class TestCaseWriter {
         lines.indented {
             val ind = test.test
             val insertionVars = mutableListOf<Pair<String, String>>()
+            val actions = getExternalServiceActions(ind)
```

Comment:
as we discussed, we handle external actions for each `RestCallAction` in terms of `EnterpriseGroup`, then reset the external between the group. 
here, you handle all external actions at the beginning of the test, right? 
it seems not correct.